src/create3-fastslam.cpp

```diff
@@ -148,15 +151,30 @@ void FastSLAMC3::control_callback(const irobot_create_msgs::msg::WheelVels& msg)
 #endif //Motion model selection
 
 void FastSLAMC3::publishSLAMOdom() {
-  RCLCPP_INFO(this->get_logger(), "Publishing robot odom delta");
+  //RCLCPP_INFO(this->get_logger(), "Publishing robot odom delta");
   auto message = nav_msgs::msg::Odometry();
   message.pose.pose.position.x = m_rob_pose_delta.x;
   message.pose.pose.position.y = m_rob_pose_delta.y;
   x_pos += m_rob_pose_delta.x;
   y_pos += m_rob_pose_delta.y;
-  RCLCPP_INFO(this->get_logger(), "Accu: x %f y %f", x_pos, y_pos);
-
+  //theta_var += m_rob_pose_delta.theta_rad
+  //RCLCPP_INFO(this->get_logger(), "Accu: x %f y %f", x_pos, y_pos);
   m_deadreckon_odom->publish(message);
```

Comment:
I missed this in my previous PR, but this should be wrapped in a conditional compile.